dist/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Restaurant</title>
+  <link rel="shortcut icon" type="image/png" href="../resources/images/restaurant-32.png?v=2"/>
+</head>
+<body>
+  <header>
+    <div id="header-caption"></div>
+  </header>
+  <main>
+    <div id="mainContent">
+      <div id="content"></div>
+    </div>
+  </main>
+  <footer id="footer"></footer>
```

Comment:
Awesome job so far.
we need to change the structure of this file.

Well done developer.... 👍 
According to the instruction on `Odin` page, we should only have `<div id="content"></div>` in side the `body` tag. so you need to remove the `header`, `the extra div`, and `footer` tags added.
![image](https://user-images.githubusercontent.com/31897434/88849163-66288600-d1e1-11ea-8414-32df71189e88.png)